mlos_core/setup.py

```diff
@@ -25,6 +30,34 @@
     warning(f"setuptools_scm failed to find git version, using version from _version.py: {e}")
 
 
+# A simple routine to read and adjust the README.md for this module into a format
+# suitable for packaging.
+# See Also: copy-source-tree-docs.sh
+# Unfortunately we can't use that directly due to the way packaging happens inside a
+# temp directory.
+# Similarly, we can't use a utility script outside this module, so this code has to
+# be duplicated for now.
+# Also, to avoid caching issues when calculating dependencies for the devcontainer,
+# we return nothing when the file is not available.
+def _get_long_desc_from_readme(base_url: str) -> dict:
+    pkg_dir = os.path.dirname(__file__)
+    readme_path = os.path.join(pkg_dir, 'README.md')
+    if not os.path.isfile(readme_path):
+        return {}
+    jsonc_re = re.compile(r'```jsonc')
+    link_re = re.compile(r'\]\(([^:#)]+)(#[a-zA-Z0-9_-]+)?\)')
```

Comment:
```suggestion
    link_re = re.compile(r'\]\(([^:#)]+)(#[\w-]+)?\)')
```

Reply:
This is the same regex used in copy-source-tree-docs.sh and I know this one works as is right now.  Retesting changes for that would be a little bit of a chore, so let's fix this later on if we run into another issue that needs it.